components/organisms/Header.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types";
+import Image from "next/image";
+import { SearchBar } from "../atoms/SearchBar";
+import { useState } from "react";
+
+export function Header(props) {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
+
+  return (
+    <div className="w-full  mt-6 ">
+      <div className="h-auto flex  md:flex-row justify-center  mt-4">
+        <div className="flex  flex-col  lg:flex-col  xl:flex-row container mx-auto justify-between relative">
+          <div className="md:h-9 h-6 mb-4">
```

Comment:
Not sure which of these it is, but the flag and search bar are not left-aligned on mobile.

<img width="420" alt="Screen Shot 2021-04-07 at 14 27 57" src="https://user-images.githubusercontent.com/2454380/113916734-53257700-97ae-11eb-9c9e-d7cdb9d109dd.png">